linker/Tests/Mono.Linker.Tests.Cases/TypeForwarding/Dependencies/ForwarderLibrary.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+
+#if INCLUDE_FORWARDERS
```

Comment:
Yes.  All of the #ifing is to handle the fact that all of the files are part of the .sln.  You can't have a TypeForwardedTo attribute defined at the same time that the type itself is defined in the same assembly.

I like having all of the files in the solution more than I dislike having the #if's...

If you can think of other ways around it, I'm open to suggestions.